### PR TITLE
UIIN-2452: Fix creating/editing/duplicating holdings/items for different member tenants (follow-up)

### PR DESCRIPTION
**Pull request by OleksandrHladchenko1**
## Approach

When user completes duplicating/editing or creating holdings/items, switch to the previous affiliation

### CODE REVIEW

**Review by mariia-aloshyna — APPROVED** [review]

File: src/Instance/InstanceDetails/MemberTenantHoldings/MemberTenantHoldings.js [hunk 1]
```diff
@@ -40,10 +40,10 @@ const MemberTenantHoldings = ({
   const { holdingsRecords, isLoading } = useInstanceHoldingsQuery(instance?.id, { tenantId: id });
   const isUserInCentralTenant = checkIfUserInCentralTenant(stripes);
 
-  const canViewHoldings = hasMemberTenantPermission('ui-inventory.instance.view', id, userTenantPermissions);
-  const canCreateItem = hasMemberTenantPermission('ui-inventory.item.edit', id, userTenantPermissions);
-  const canCreateHoldings = hasMemberTenantPermission('ui-inventory.holdings.edit', id, userTenantPermissions);
-  const canViewItems = hasMemberTenantPermission('ui-inventory.instance.view', id, userTenantPermissions);
+  const canViewHoldings = hasMemberTenantPermission('ui-inventory.holdings.create', id, userTenantPermissions);
```
Comment (BogdanDenis) [inline src/Instance/InstanceDetails/MemberTenantHoldings/MemberTenantHoldings.js:43]: Please also update variable names

**Review by BogdanDenis — CHANGES_REQUESTED** [review]

File: src/Instance/InstanceDetails/MemberTenantHoldings/MemberTenantHoldings.js (hunk 1 above)
Comment (OleksandrHladchenko1) [reply src/Instance/InstanceDetails/MemberTenantHoldings/MemberTenantHoldings.js:43]: Hi @BogdanDenis To view and create holdings we need "View, create holdings" permission (ui-inventory.holdings.create).  **canViewHoldings** is used for "View holdings" button. **canCreateHoldings** is used for "Add holdings" button

**Review by UladzislauKutarkin — APPROVED** [review]

File: src/Instance/InstanceDetails/MemberTenantHoldings/MemberTenantHoldings.js (hunk 1 above)
Comment (BogdanDenis) [reply src/Instance/InstanceDetails/MemberTenantHoldings/MemberTenantHoldings.js:43]: It's confusing to me why there are two variables that are checking for the same permission and they are called differently.
Additionally, from variable name `canViewHoldings` I expect it to be true only when user has view permissions, but not edit or create.

Comment (mariia-aloshyna) [reply src/Instance/InstanceDetails/MemberTenantHoldings/MemberTenantHoldings.js:43]: @BogdanDenis I was also confused about the fact that to view holdings you actually need to check for View and **create** permission, we discussed it with PO and figured out that there is no separate permission to just view holdings. I suggest renaming this variable to `canViewAndCreateHoldings` and using it in two places

Comment (BogdanDenis) [reply src/Instance/InstanceDetails/MemberTenantHoldings/MemberTenantHoldings.js:43]: @OleksandrHladchenko1 please review comment from @mariia-aloshyna

**Comment by OleksandrHladchenko1** [comment]
@BogdanDenis please, review one more time

**Review by BogdanDenis — APPROVED** [review]

**Pull request merged**